Panorama enhancement - add target arg (#18537)
demisto/content: Packs/PAN-OS/ReleaseNotes/1_14_3.md

#### Integrations
##### Palo Alto Networks PAN-OS

> - Added support for the *target* argument in the following commands. The target argument is supported only by panorama op type commands. (not including logs, commits, etc)

Reviewer: ```suggestion
- Added support for the *target* argument in the following commands. The target argument is supported only in operational type commands. Meaning, you can not use it with commit, logs, PCAP commands.
```

  - ***pan-os-get-url-category***
  - ***pan-os-device-reboot***
  - ***pan-os-list-rules***
  - ***pan-os-show-device-version***
  - ***pan-os-download-latest-content-update***
  - ***pan-os-content-update-download-status***
  - ***pan-os-install-latest-content-update***
  - ***pan-os-content-update-install-status***
  - ***pan-os-check-latest-panos-software***
  - ***pan-os-download-panos-version***
  - ***pan-os-download-panos-status***
  - ***pan-os-install-panos-version***
  - ***pan-os-install-panos-status***
  - ***pan-os***

> - Added support for human-readable output for the ***pan-os-check-latest-panos-software*** command to present the top 5 most recent software releases.

Reviewer: ```suggestion
- Added in the ***pan-os-check-latest-panos-software*** command the top 5 most recent software releases.
```
